### ISSUE TITLE
no json_* options for exec input plugin

### ISSUE THREAD

**Issue by amarao** [influxdata/telegraf#4708]
### Relevant telegraf.conf:
Configuration from example in documentation
(https://github.com/influxdata/telegraf/blob/master/docs/DATA_FORMATS_INPUT.md#json)
```
[[inputs.exec]]
  ## Commands array
  commands = ["/usr/bin/mycollector --foo=bar"]

  ## measurement name suffix (for separating different commands)
  name_suffix = "_mycollector"

  ## Data format to consume.
  ## Each data format has its own unique set of configuration options, read
  ## more about them here:
  ## https://github.com/influxdata/telegraf/blob/master/docs/DATA_FORMATS_INPUT.md
  data_format = "json"

  ## List of tag names to extract from top-level of JSON server response
  tag_keys = [
    "my_tag_1",
    "my_tag_2"
  ]

  ## List of field names to extract from JSON and add as string fields
  # json_string_fields = []

  ## gjson query path to specify a specific chunk of JSON to be parsed with
  ## the above configuration. If not specified, the whole file will be parsed
  # json_query = ""

  ## holds the name of the tag of timestamp
  json_time_key = "b_time"

  ## holds the format of timestamp to be parsed
  json_time_format = "02 Jan 06 15:04 MST"
```
### System info:
telegraf                              1.7.0-1
Ubuntu 18.04

UPD:  telegraf 1.7.4-1 have the same problem

### Steps to reproduce:
Write config. Run --test
### Expected behavior:
Work
### Actual behavior:
E! Error parsing /etc/telegraf/telegraf.conf, line 37: field corresponding to `json_time_format' is not defined in `*exec.Exec'

### Additional info:
The same error is for all json_* options (`json_string_fields`, etc)

**Comment by kurbeco** [comment]
Possible link to https://github.com/influxdata/telegraf/issues/4705

**Comment by danielnelson** [comment]
These new options are only available in 1.8, currently the latest version is 1.8.0-rc1:

https://github.com/influxdata/telegraf/releases/tag/1.8.0-rc1
